apps/mobile/src/components/Post/PostInput.tsx

```diff
@@ -53,6 +53,7 @@ export function PostInput({ value, onChange, tokenRef }: Props) {
         maxLength={MAX_LENGTH}
         autoCapitalize="none"
         autoComplete="off"
+        keyboardAppearance={colorScheme === 'dark' ? 'dark' : 'light'}
```

Comment:
might just be able to pass in colorScheme directly? seems like the only 2 values are light and dark
<img width="437" alt="CleanShot 2023-08-11 at 14 13 54@2x" src="https://github.com/gallery-so/gallery/assets/80802871/668604b2-cbca-4195-aaad-46926cb890ba">

Reply:
Good one! Updated it to pass directly